_dependencies/tasks/main.yml

```diff
@@ -1,7 +1,19 @@
 ---
 
-- name: Force include of group_vars on localhost (no inventory yet, so cannot import automatically)
-  include_vars: { dir: "{{ playbook_dir }}/group_vars/{{ clusterid }}" }
+- block:
+  - name: Force include of group_vars on localhost (no inventory yet, so cannot import automatically)
+    include_vars: { dir: "{{ playbook_dir }}/group_vars/{{ clusterid }}" }
+    when: 
+      - cluster_vars_format == 'native'

```

Comment:
could we add `or cluster_vars_format is undefined` instead and avoid explicitly defining here: https://github.com/sky-uk/clusterverse/pull/42/files#diff-62bdd23f6512b7480bd90b92a2c7b476R31 ?

Reply:
what the reason to do that ?

Reply:
thanks added